File: Rucksack alarm mit Sterung/rucksack_alarm_empfaenger.py
def on_received_number(receivedNumber):
    global modus
    if receivedNumber == 13:
        modus = 1
    if receivedNumber == 12:
        modus = 0
radio.on_received_number(on_received_number)

modus = 0
radio.set_group(1)

def on_forever():
    if modus == 0:
        if input.light_level() <= 100:
            music.stop_all_sounds()
        if input.light_level() > 100:
            music.play_tone(932, music.beat(BeatFraction.WHOLE))
basic.forever(on_forever)

File: Rucksack alarm mit Sterung/rucksack_alarm_empfaenger.js
radio.onReceivedNumber(function (receivedNumber) {
    if (receivedNumber == 13) {
        modus = 1
    }
    if (receivedNumber == 12) {
        modus = 0
    }
})
let modus = 0
radio.setGroup(1)
basic.forever(function () {
    if (modus == 0) {
        if (input.lightLevel() <= 100) {
            music.stopAllSounds()
        }
        if (input.lightLevel() > 100) {
            music.playTone(932, music.beat(BeatFraction.Whole))
        }
    }
})
